src/reference/changes.rst

```diff
@@ -36,6 +36,95 @@ Cylc 8.5
    :cylc-rose: `1.5 <https://github.com/cylc/cylc-rose/blob/master/CHANGES.md>`__
    :rose: `2.4 <https://github.com/metomi/rose/blob/master/CHANGES.md>`__
 
+
+Task Matching
+^^^^^^^^^^^^^
+
+Various Cylc commands (e.g. ``cylc trigger``, ``cylc set`` and ``cylc hold``)
+allow us to select the tasks we want to operate on using cycle points,
+family names or globs, e.g:
+
+.. code-block:: bash
+
+   # trigger active tasks in cycle "2000"
+   cylc trigger workflow//2000
+
+   # trigger active tasks in "FAMILY" in cycle "2000"
+   cylc trigger workflow//2000/FAMILY
+
+   # trigger all tasks in cycle "2000" with names starting with "foo"
+   cylc trigger workflow//2000/foo*
+
+However, Cylc would only operate on :term:`active tasks <active task>`.
```

Comment:
This isn't very clear unless you carefully read the entire section, better to make it stand alone.
```suggestion
However, prior to 8.6 family names and globs would only match :term:`active tasks <active task>`.
```